src/styles/common.scss

```diff
@@ -299,18 +299,36 @@
     text-wrap: balance;
 }
 
+.floatingCardBackground {
+    /* negate padding from <main> element */
```

Comment:
I think this comment is not necessary anymore because `var(--pagePaddingHoriz)` explains it.